content/_help/verify-your-identity/phone-number-and-phone-plan-in-your-name._es.md

```diff
@@ -1,19 +1,15 @@
 ---
-title: "Número de teléfono y plan telefónico a su nombre "
+title: Verifique su número de teléfono
 category: verify-your-identity
-permalink: /es/help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
+permalink: /es/help/verify-your-identity/phone-number/
+redirect_from:
+  - /es/help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
 order: 5
 ---
 Para completar con éxito la verificación de la identidad, debe proporcionar un número de teléfono con domicilio en EE.UU. a su nombre en el plan telefónico. 
```

Comment:
"su nombre en el plan telefónico" translates for me as "your name on the phone plan". Did we expect to update this as well?

Reply:
Another good catch. Thank you! Missed having this line translated earlier, so I'll commit the changes shortly.